lib/xml.js

```diff
@@ -73,22 +73,22 @@ var tag = function (name, prefix, value, attribs) {
 };
 
 var cdata = function(str) {
-  if (common.isString(str) && str.trim().length > 0) {
+  if (!_.isUndefined(str) && !_.isNull(str)) {
```

Comment:
I call the cdata function for custom fields, it used to be called only for userFields.
I would suggest maybe calling it systematically and transforming it like this :
```
var cdata = function(value) {
    if (!_.isUndefined(value) && !_.isNull(value)) {
        return '<![CDATA[' + value + ']]>';
    } else {
        return value;
    }
};
```